geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/data/RedisHash.java

```diff
@@ -84,21 +83,8 @@ public RedisHash() {}
   @Override
   public synchronized void toData(DataOutput out, SerializationContext context) throws IOException {
     super.toData(out, context);
+    hash.toData(out);
```

Comment:
I think this needs to come after the size is written, to match the order in `fromData()`, or the size needs to be written and read as part of the Hash to/fromData methods.

Reply:
I messed up and meant to copy the line that wrote the size to hash.toData. Serialization is working now